examples/synthesis_json.py

```diff
@@ -91,27 +91,28 @@ def transform_distribution(data, datamin=None, datamax=None):
     '''
     data_dict = OrderedDict()
 
-    if data["type"] == 'norm':
-        data_dict.update({"type": "normal"})
-        data_dict.update({"mu": data["params"][0]})
-        data_dict.update({"sigma": data["params"][1]})
+    if data["type"] == 'uniform':
+        data_dict.update({"type": "uniform",
+                          "min": np.min(data["params"]),
+                          "max": np.max(data["params"])})
+        return data_dict
```

Comment:
The logic here is that min/max are the parameters of the uniform distribution, so we ignore datamin/datamax.